src/extent.c

```diff
@@ -965,10 +1016,15 @@ extent_recycle(tsdn_t *tsdn, arena_t *arena, extent_hooks_t **r_extent_hooks,
  * memory, in which case *zero is toggled to true.  arena_extent_alloc() takes
  * advantage of this to avoid demanding zeroed extents, but taking advantage of
  * them if they are returned.
+ *
+ * The caller can specify szind == NSIZES to indicate "don't know" (forgoing the
```

Comment:
Yeah. We care about szind for sized-alloc-region extents, but for all others the extent code is mostly agnostic about it.